setup-env

```diff
@@ -0,0 +1,37 @@
+# This script sets up an environment suitable for development.
+#
+# Usage:
+#
+#   $ source setup-env
+#
```

Comment:
This is important.. the idea is that, when you want to start working on this project, you should just type
```sh
$ source setup-env
```
and this script will take care of creating a virtual environment and installing dependencies into it so you can just edit/lint/test/run the code in your shell.